lib/matplotlib/_api/__init__.py

```diff
@@ -174,12 +175,19 @@ def check_shape(shape, /, **kwargs):
             )
 
 
-def check_getitem(mapping, /, **kwargs):
+def check_getitem(
+        mapping, /, _error_cls=ValueError, **kwargs
+    ):
```

Comment:
This fits in a line.
```suggestion
def check_getitem(mapping, /, _error_cls=ValueError, **kwargs):
```

Reply:
And this violates E125, which is currently missing from `ruff`, but we shouldn't introduce more linting issues for the future.